code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -30,34 +44,39 @@
 			eye_blurry = max(2, eye_blurry)
 			stuttering = max(stuttering, 5)
 		if(60 to 79)
-			if(shock_stage == 60 && !lying)
-				emote("me", 1, " is having trouble standing.")
 			eye_blurry = max(2, eye_blurry)
 			stuttering = max(stuttering, 5)
 			if(prob(2))
+				if(!lying)
+					emote("me", 1, " is having trouble standing.")
 				to_chat(src, "<span class='danger'>[pick("The pain is excruciating", "Please, just end the pain", "Your whole body is going numb")]!</span>")
-				KnockDown(20)
+				KnockDown(5)
```

Comment:
This is a big change to how pain is processed, not just halloss. Will require some testing.